.github/workflows/tox.yml

```diff
@@ -8,12 +8,29 @@ on:
      - master
 
 jobs:
+  build36:
```

Comment:
Wish we could remove the "steps" duplication in build36 and build.
Unfortunately, [Github still doesn't support YAML anchors](https://github.com/actions/runner/issues/1182).